c/driver/postgresql/postgresql_test.cc

```diff
@@ -432,11 +429,14 @@ TEST_F(PostgresConnectionTest, GetObjectsGetAllFindsForeignKey) {
       << "expected 1 constraint on adbc_fkey_test table, found: "
       << table->n_table_constraints;
 
+  const std::string version =
+      adbc_validation::GetDriverVendorVersion(&connection, &error);
+  // ASSERT_EQ(ADBC_STATUS_OK, &error);
```

Comment:
This isn't right but I couldn't find the proper pattern to use. Seems like all of the other methods in util that could fail are `void` and raise directly. Should I make this function also void and just mutate a buffer, or is there a simpler pattern for this?